default_context.go

```diff
@@ -161,11 +161,9 @@ func (d *DefaultContext) Render(status int, rr render.Renderer) error {
 // binder is "http://www.gorillatoolkit.org/pkg/schema".
 func (d *DefaultContext) Bind(value interface{}) error {
 	ct := strings.ToLower(d.Request().Header.Get("Content-Type"))
-	cts := strings.Split(ct, ";")
-	for _, c := range cts {
-		if b, ok := binders[strings.TrimSpace(c)]; ok {
-			return b(d.Request(), value)
-		}
+	cts := strings.Split(ct, ";")[0]
```

Comment:
Splitting on an empty string returns a slice that has 1 empty string element in it:

https://play.golang.org/p/_y55t5_kC9

Still, I'll wrap it anyway, just to prevent future peeps from questioning at it.